src/main/resources/pl/damianszczepanik/jenkins/buildhistorymanager/model/Rule/help-matchAtMost.html

```diff
@@ -2,12 +2,23 @@
 <p>The rule will stop being processed after a certain number of matched builds.
     Once this condition is met, only the other rules will continue to be processed.</p>
 
-<p><b>Use cases</b></p>
-<p>This feature provides the option to control the number of builds the rule can process. It may save a significant amount of time when the build history is long.</p>
+<p>Internally and in the documentation, this value is called <code>matchAtMost</code>.</p>
```

Comment:
from the user perspective this is valuable how to use/configure and not how this is named internally